resources/views/queue-monitor/jobs.blade.php

```diff
@@ -0,0 +1,141 @@
+                        <a class="btn-sm btn-danger ml-1 mr-1 forget_all d-none" onclick="forget_job(this)"
+                           href="javascript:void(0)">Forget All</a>
+                    </div>
+                </div>
+                <!-- /.card-header -->
+                <div class="card-body">
+                    <div class="dataTables_wrapper dt-bootstrap4">
+                        <div class="row">
+                            <div class="col-sm-12">
+                                <table class="table table-bordered table-striped dataTable" role="grid">
+                                    <thead>
+                                    <tr role="row">
+                                        <th>ID</th>
+                                        <th>Queue</th>
+                                        <th>Job</th>
+                                        <th>Error</th>
+                                        <th>Created/Failed</th>
+                                        <th>Action</th>
+                                    </tr>
+                                    </thead>
+                                    <tbody>
+                                    </tbody>
+                                    <tfoot>
+                                    </tfoot>
+                                </table>
+                            </div>
+                            <!-- /.card-body -->
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+@stop
+
+@section('js')
+    <script type="text/javascript">
+        let queue_filter = $("#queue_filter");
+        let table = {};
+
+        function initializeDataTable(queue_filter) {
+            table = $('.table').DataTable({
+                processing: true,
+                serverSide: true,
+                searchDelay: 800,
+                pageLength: 25,
+                deferRender: true,
+                ajax: {
+                    url: "{{ route('admin.queue-monitor.jobs') }}",
+                    data: {
+                        filter: queue_filter,
+                    },
+                },
+                columns: [
+                    {data: 'id', name: 'id'},
+                    {data: 'queue', name: 'queue'},
+                    {data: 'payload', name: 'payload'},
+                    {data: 'exception', name: 'exception'},
+                    {data: 'time', name: 'time', orderable: false, searchable: false},
+                    {data: 'action', name: 'action', orderable: false, searchable: false},
+                ],
+                "order": [[0, "desc"]]
+            });
+        }
+
+        // initialize datatable
+        initializeDataTable(queue_filter.val());
+
+        // if queue filter changes
+        queue_filter.on("change", function () {
+            reInitializeDataTable();
+            $(".retry_all,.forget_all").toggleClass("d-none");
```

Comment:
Could add a space after comma for the selectors. Try it to see if that still works.

Reply:
Yes, it worked. Added space after comma.